readthedocs.yml

```diff
@@ -1,2 +1,9 @@
-conda:
-    file: environment.yml
\ No newline at end of file
+version: 2
+python:
+   version: 3.6
```

Comment:
Maybe 3.8 instead, or is there some problem with the latest release?

Reply:
Sure, I think use 3.8 should be fine. 😃